posthog/temporal/tests/utils/models.py

```diff
@@ -30,7 +30,12 @@ async def acreate_batch_export(team_id: int, interval: str, name: str, destinati
 async def adelete_batch_export(batch_export: BatchExport, temporal_client: temporalio.client.Client) -> None:
     """Async delete a BatchExport and its underlying Schedule."""
     handle = temporal_client.get_schedule_handle(str(batch_export.id))
-    await handle.delete()
+
+    try:
+        await handle.delete()
+    except temporalio.service.RPCError:
+        # This means the schedule was already deleted, so we can continue
```

Comment:
Is that really all this error could mean here? What about a temporary issue talking to Temporal, or something like that? It's a shame it's not more explicit.

Reply:
This one, in particular, is a test utility. So I would not worry about this one too much.